plume-cli/src/users.rs

```diff
@@ -76,3 +91,15 @@ fn new<'a>(args: &ArgMatches<'a>, conn: &Connection) {
     ).expect("Couldn't save new user")
     .update_boxes(conn).expect("Couldn't update ActivityPub informations for new user");
 }
+
+fn reset_password<'a>(args: &ArgMatches<'a>, conn: &Connection) {
+    let username = args.value_of("name").map(String::from).unwrap_or_else(|| super::ask_for("Username"));
+    let password = args.value_of("password").map(String::from).unwrap_or_else(|| {
+        print!("Password: ");
+        io::stdout().flush().expect("Couldn't flush STDOUT");
+        rpassword::read_password().expect("Couldn't read your password.")
+    });
+    let user = User::find_by_name(conn, &username, Instance::get_local(conn).expect("Failed to get local instance").id)
+        .expect("Failed to get user");
```

Comment:
shouldn't we do this bit before asking for a new password? might be a bit frustrating for folks who are prone to typis